.github/workflows/error-ref-publisher.yaml

```diff
@@ -0,0 +1,51 @@
+name: Error Reference Publisher
+on:
+  push:
+    branches:
+      - 'err-ref-ci'
+
+jobs:
+  error-ref:
+    name: Error codes utility
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{ secrets.GO_VERSION }}
+
+      - name: Run utility
+        run: |
+          go run github.com/layer5io/meshkit/cmd/errorutil -d . analyze --skip-dirs meshery
```

Comment:
You could update the `make` command and use them here @DelusionalOptimist

Reply:
Done :+1: